requirements_doc.txt

```diff
@@ -0,0 +1,2 @@
+sphinx
+sphinx-rtd-theme
```

Comment:
I *really* dislike the RTD theme. Can you please remove?

Reply:
Also, note that the dependencies are listed/installed here: https://github.com/conda-forge/conda-forge.github.io/blob/master/.ci_scripts/update_docs#L13 This file can probably be removed

Reply:
I added all required packages to the conda command. The requirements file I still need for rendering the preview on RTD. I will remove it right before the PR gets merged.